chainer/variable.py

```diff
@@ -215,15 +215,15 @@ def to_cpu(self):
         if self._grad is not None:
             self._grad = cuda.to_cpu(self._grad)
 
-    def to_gpu(self, device=None):
+    def to_gpu(self, device_id=None):
         """Copies the data and gradient arrays to specified GPU.
 
         Args:
-            device: Target device specifier. If omitted, the current device is
-                used.
+            device_id (int): The target device ID. If omitted, the current
+                device is used.
 
         """
-        with cuda.get_device_from_id(device.id if device else None):
+        with cuda.get_device_from_id(device_id):
```

Comment:
I found `get_device` function are still remaining in `to_gpu()` method defined in `chainer/cuda.py`, so I'll replace them with `get_device_from_id` and `get_device_from_array`. Then the below two `to_gpu` will also be replaced with `to_gpu(array, device_id)`.